<?xml version="1.0"?>
<launch>
    <node pkg="stereo_ros" type="stereo_ros" name="stereo_ros" output="screen">
        <param name="view_input" type="bool" value="false" />
        <param name="view_output" type="bool" value="false" />
        <param name="save_or_not" type="bool" value="false" />
        <param name="save_dir" type="str" value="$(find stereo_ros)/output/" />
        <param name="approximate_sync" type="bool" value="false" />
        <param name="queue_size" type="int" value="1" />

        <param name="scale" type="double" value="1.0" />
        <param name="offset_x" type="int" value="0" />
        <param name="offset_y" type="int" value="0" />
        <param name="crop_h" type="int" value="576" />
        <param name="crop_w" type="int" value="1024" />
        <param name="p1" type="int" value="10" />
        <param name="p2" type="int" value="120" />
        <param name="disp_size" type="int" value="64" />
        <param name="uniqueness" type="double" value="0.95" />
        <param name="num_paths" type="int" value="8" />
        <param name="min_disp" type="int" value="0" />
        <param name="LR_max_diff" type="int" value="1" />
        <param name="cen" type="int" value="0" />

        <remap from="/left/image_rect" to="/zed2/zed_node/left/image_rect_color"/>
        <remap from="/right/image_rect" to="/zed2/zed_node/right/image_rect_color"/>
    </node>
</launch>